docs/unified-test-documentation/dasharo-compatibility/power-after-fail.md

```diff
@@ -0,0 +1,174 @@
+# Dasharo Compatibility: Power State After Power Failure
+
+## PSF001.001 Check Power State After Power Failure default state (firmware)
+
+**Test description**
+
+This BIOS setup feature allows users to determine the system's power state
+after a power failure. Users can choose between restoring the previous power
+state or always returning to a powered off state. This test ensures that the
+option is present, and the default state of this option after flashing is
+correct.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI Setup
+    Menu.
+1. Enter the `Dasharo System Features` menu using the arrow keys and Enter.
+1. Enter the `Power Management Options` submenu.
+1. Verify the `Power state after power failure` field.
+
+**Expected result**
+
+The `Power state after power failure` field should inform that the current
+state of
+Secure Boot is `Powered Off`.
```

Comment:
Copypaste error? :)

Reply:
Yes indeed, resolved in  [recent commit](https://github.com/Dasharo/docs/commit/f401a74c4b776a6989141b7a340943a1ddfa0294)